lago/plugins/vm.py

```diff
@@ -483,6 +483,15 @@ def all_ips(self):
                     ips.append(str(ip))
         return ips
 
+    def ips_in_net(self, net_name):
+        ips = []
+        net = self.virt_env.get_net(name=net_name)
+        mapping = net.mapping()
+        for hostname, ip in mapping.items():
+            if hostname.startswith(self.name()):
```

Comment:
Why not use `hostname == self.name()` ?
The current implementation can yield unwanted results.
For example, if the mapping is as follows:
```
m = {
       'lago-basic-suite-master-host1': '1.2.3.1',
       'lago-basic-suite-master-host11': '1.2.3.3'
}
```
`lago-basic-suite-master-host1` will get `lago-basic-suite-master-host11` ip.

Reply:
I now see that `==` will no work because we are adding to the domain name the interface name.
So just add '-' to `self.name()` in line 491.

Reply:
Done.